graylog2-server/src/main/java/org/graylog2/indexer/IndexSetValidator.java

```diff
@@ -120,9 +120,10 @@ public Violation validateRotation(RotationStrategyConfig rotationStrategyConfig)
                 return Violation.create(f("%s <%s> is shorter than %s <%s>", INDEX_LIFETIME_MAX, config.indexLifetimeMax(),
                         INDEX_LIFETIME_MIN, config.indexLifetimeMin()));
             }
-            if (leeway.toStandardSeconds().isLessThan(elasticsearchConfiguration.getTimeSizeOptimizingRotationPeriod().toStandardSeconds())) {
+
+            if (leeway.toStandardSeconds().isLessThan(elasticsearchConfiguration.getTimeSizeOptimizingRotationFixedLeeway().toStandardSeconds())) {
```

Comment:
i would keep this validation and add a new separate fixed leeway validation (which is off by default (see above comment))